docs/examples/1_presentation/example_1_presentation_2_heartbeat_getinfo_udp.cpp

```diff
@@ -156,17 +162,23 @@ TEST_F(Example_12_PosixUdpPresentation, heartbeat_and_getInfo)
     });
 
     // Bring up 'GetInfo' server.
-    // TODO: Replace with service server when it will be available.
-    state.get_info_.setName("org.opencyphal.example_12_posix_udp_presentation");
-    state.get_info_.makeRxSession(*state.transport_);
-    state.get_info_.makeTxSession(*state.transport_);
-
-    // Main loop.
     //
-    CommonHelpers::runMainLoop(executor_, startup_time_ + run_duration_ + 500ms, [&](const auto now) {
+    using GetInfo_1_0 = uavcan::node::GetInfo_1_0;
+    const std::string node_name{"org.opencyphal.Ex_1_Pres_2_HB_GetInfo_UDP"};
+    std::copy_n(node_name.begin(), std::min(node_name.size(), 50UL), std::back_inserter(state.get_info_response.name));
+    //
+    auto maybe_get_info_srv = presentation.makeServer<GetInfo_1_0>([&state](const auto& arg, auto continuation) {
```

Comment:
Creating of standard fixed port service is super simple:
- Request, Response, their pord ids, extent bytes and de/serialization - all done automatically.
- Reply continuation could be provided right on during server creation (or later if needed).
- As with `Subscriber` api, you can always pass instead of lambda some sort of "adapter" functor, which may provide necessary storing of request and their continuations, as well as peek/pop api - not responsibility of `Server`.